cogs/giftdrop.py

```diff
@@ -71,19 +68,22 @@ async def on_message(self, message: discord.Message):
         if message.channel.id not in self.bot.config.get("drop_channels", []): return
         self.users_last_channel[message.author.id] = {'name': message.channel.name, 'id': message.channel.id}
 
+        # Remove markdown
+        for pattern in ["***", "**", "*", "__", "~~", "||", "`", "```", ">"]:
```

Comment:
Don't think you need to include `*` and back-tick once without, `**`, `***`, or triple back-tick as they'll already be covered by a single character filter.